.github/workflows/dependencies.yml

```diff
@@ -0,0 +1,75 @@
+name: "Verify dependency chain"
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  dependency-change-verification:
+    runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.changeDetection.outputs.should-run }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v44
+      - name: Verify build.gradle changed
+        id: changeDetection
+        env:
+          ALL_CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
+        run: |
+          GRADLE='build.gradle'
+          for file in ${ALL_CHANGED_FILES}; do
+            if [[ "$file" == *"$GRADLE" ]]; then
+              echo "$file was changed"
+              echo "should-run=true" >> $GITHUB_OUTPUT 
+            fi
+          done
+
+  verify-dependencies:
+    runs-on: ubuntu-latest
+    needs: dependency-change-verification
+    if: needs.dependency-change-verification.outputs.changed == 'true'
```

Comment:
Wouldn't it be easier to use something like this?

```yaml
on:
  push:
    paths:
      - build.gradle
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
Ohh that is indeed easier! I'll check it out, thanks!